contrib/kube-prometheus/manifests/thanos/prometheus-self.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
can we move these manifests to `experimental/` to match the changes in [this](https://github.com/coreos/prometheus-operator/pull/1206) PR?